spring-pulsar-docs/src/main/antora/modules/ROOT/pages/reference/topic-resolution.adoc

```diff
@@ -48,6 +48,20 @@ record Foo(String value) {
 }
 ----
 
+Property placeholders and SpEL expressions are supported in the `@PulsarMessage` annotation,
+for example:
+[source,java,indent=0,subs="verbatim"]
+----
+@PulsarMessage(topic = "${app.topics.foo}")
+record Foo(String value) {
+}
+----
```

Comment:
Since we say PPH and SpEL are supported, might be helpful to add another example w/ a SpEL expression.